docs/operators.md

```diff
@@ -1,64 +1,64 @@
-The following table lists the ONNX operators supported by each of the available backends.
+The following table lists the [ai.onnx](https://github.com/onnx/onnx/blob/master/docs/Operators.md) operators supported by each of the available backends.
```

Comment:
There is a slight inconsistency in the URL mentioned here and that used to link to the specific operators below- 'master' branch vs 'rel1.2.3' (which should be OpSet v7). @fs-eire  - which one should we keep ?